### PR TITLE
chore(version): v1 Release

### PR DESCRIPTION
**Pull request by dnys1**
Stable v1 release! 🎉

Updated-Components: amplify_lints, Amplify Flutter, Amplify Dart, Amplify UI, DB Common, Secure Storage, AWS Common, Smithy, Worker Bee

### CODE REVIEW

File: packages/analytics/amplify_analytics_pinpoint_dart/lib/src/version.dart [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // Generated code. Do not modify.
-const packageVersion = '0.2.0+1';
+const packageVersion = '0.2.0+2';
```
Comment (HuiSF) [inline packages/analytics/amplify_analytics_pinpoint_dart/lib/src/version.dart:2]: Sanity check - Is this used for generating user-agent? If yes, should this be located under `amplify_analytics_pinpoint` with version `1.0.0`?

File: packages/amplify/amplify_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+## 1.0.0
+
+We are thrilled to release version 1.0 of the Amplify Flutter libraries!
+
+This release introduces a full rewrite of the Amplify libraries in Dart, allowing you to build AWS-powered
+applications on every platform Flutter supports (iOS, Android, Web, macOS, Windows, and Linux).
+
+If you've been using v0 in production, we believe you can now safely migrate to v1. Check out our
```
Comment (ragingsquirrel3) [inline packages/amplify/amplify_flutter/CHANGELOG.md:8]: nit/question: Should all these be changed to "in production, you can now ..." to remove "we believe." ?

File: packages/storage/amplify_storage_s3/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+more information.
+
+As always, you can find us on [GitHub](https://github.com/aws-amplify/amplify-flutter/) and 
+[Discord](https://discord.gg/jWVbPfC) to answer any questions.
+
+### New Features
+
+* In-flight download and upload operations are now fully controllable.
+    * NOTE: Downloading and uploading in the background is not yet supported.
+* The `list` API now supports pagination
```
Comment (HuiSF) [inline packages/storage/amplify_storage_s3/CHANGELOG.md:19]:
```suggestion
* The `list` API now supports pagination.
```

File: packages/storage/amplify_storage_s3/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+* The new APIs `uploadData` and `downloadData` provide flexibility when dealing with data stored in memory.
+    * `uploadData` supports uploading from various data formats such as raw bytes, JSON, strings, Data URLs, and more.
+* The new API `getProperties` allows retrieving S3 object metadata without downloading the object. 
+* New `copy`, `move`, and `removeMany` APIs now support better control of your files.
+* Prefix customization is now supported
```
Comment (HuiSF) [inline packages/storage/amplify_storage_s3/CHANGELOG.md:24]:
```suggestion
* Prefix customization is now supported.
```

**Review by HuiSF — APPROVED** [review]

File: packages/analytics/amplify_analytics_pinpoint_dart/lib/src/version.dart (hunk 1 above)
Comment (dnys1) [reply packages/analytics/amplify_analytics_pinpoint_dart/lib/src/version.dart:2]: It's not used in the user agent but in the event's sdkVersion field. @fjnoyp please confirm or adjust.

We do want to support both Dart and Flutter usages so it's okay for this package to embed it's version as long as the Flutter package overrides it if needed.

File: packages/amplify/amplify_flutter/CHANGELOG.md (hunk 2 above)
Comment (dnys1) [reply packages/amplify/amplify_flutter/CHANGELOG.md]: Done 👍

File: packages/amplify/amplify_flutter/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,17 @@
+## 1.0.0
+
+We are thrilled to release version 1.0 of the Amplify Flutter libraries!
+
+This release introduces a full rewrite of the Amplify libraries in Dart, allowing you to build AWS-powered
```
Comment (abdallahshaban557) [inline packages/amplify/amplify_flutter/CHANGELOG.md:5]: allowing you to build AWS cloud-connected applications

File: packages/storage/amplify_storage_s3/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,28 @@
+## 1.0.0
+
+We are thrilled to release version 1.0 of the Amplify Flutter libraries!
+
+This release introduces a full rewrite of the Amplify libraries in Dart, allowing you to build AWS-powered
```
Comment (abdallahshaban557) [inline packages/storage/amplify_storage_s3/CHANGELOG.md:5]: allowing you to build AWS cloud-connected applications

File: packages/storage/amplify_storage_s3/CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,28 @@
...
+
+If you've been using v0 in production, you can now safely migrate to v1. Check out our 
+[migration guide](https://docs.amplify.aws/lib/project-setup/upgrade-guide/q/platform/flutter/) for 
+more information.
+
+As always, you can find us on [GitHub](https://github.com/aws-amplify/amplify-flutter/) and 
+[Discord](https://discord.gg/jWVbPfC) to answer any questions.
+
+### New Features
+
```
Comment (abdallahshaban557) [inline packages/storage/amplify_storage_s3/CHANGELOG.md:16]: can we add a header here saying "Storage Category"

File: packages/storage/amplify_storage_s3/CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,28 @@
...
+* In-flight download and upload operations are now fully controllable.
+    * NOTE: Downloading and uploading in the background is not yet supported.
+* The `list` API now supports pagination.
+* The new APIs `uploadData` and `downloadData` provide flexibility when dealing with data stored in memory.
+    * `uploadData` supports uploading from various data formats such as raw bytes, JSON, strings, Data URLs, and more.
+* The new API `getProperties` allows retrieving S3 object metadata without downloading the object. 
+* New `copy`, `move`, and `removeMany` APIs now support better control of your files.
+* Prefix customization is now supported.
+
```
Comment (abdallahshaban557) [inline packages/storage/amplify_storage_s3/CHANGELOG.md:25]: can we also add here a section to link to all the bug fixes we are addressing with this release? I think we had a label we were using already.

**Review by Jordan-Nelson — APPROVED** [review]

**Review by HuiSF — DISMISSED** [review]

**Review by abdallahshaban557 — DISMISSED** [review]
LGTM

**Review by Jordan-Nelson — APPROVED** [review]

**Review by HuiSF — APPROVED** [review]

**Review by Jordan-Nelson — APPROVED** [review]

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Pull request merged**